tests/test_backend.py

```diff
@@ -111,6 +126,546 @@ def test_file_formats_merging(self, multi_backend_connection, config, backend1,
         }
 
 
+class TestAggregatorSecondaryServices:
+
+    def test_service_types_simple(self, multi_backend_connection, config, backend1, backend2, requests_mock):
+        """Given 2 backends and only 1 backend has a single service type, then the aggregator
+            returns that 1 service type's metadata.
+        """
+        single_service_type = {
+            "WMTS": {
+                "configuration": {
+                    "colormap": {
+                        "default": "YlGn",
+                        "description":
+                        "The colormap to apply to single band layers",
+                        "type": "string"
+                    },
+                    "version": {
+                        "default": "1.0.0",
+                        "description": "The WMTS version to use.",
+                        "enum": ["1.0.0"],
+                        "type": "string"
+                    }
+                },
+                "links": [],
+                "process_parameters": [],
+                "title": "Web Map Tile Service"
+            }
+        }
+        requests_mock.get(backend1 + "/service_types", json=single_service_type)
+        requests_mock.get(backend2 + "/service_types", json=single_service_type)
```

Comment:
comment says

> Given 2 backends and only 1 backend has a single service type

but backend2 also lists a service type

Reply:
Good catch. I will correct it.

Reply:
Now corrected.